scripts/praise.coffee

```diff
@@ -50,18 +50,19 @@ module.exports = (robot) ->
       intervals.splice(i, 1)
 
   # Send a praise/shame message and @-mention the given user
-  sendMessage = (msg, user, shame = false) ->
+  sendMessage = (msg, user, shame = false, reason = '') ->
     if shame
-      msg.send "#{user} SHAME! :bell:"
+      msg.send "#{user} SHAME! #{reason} :bell:"
```

Comment:
When reason is empty the double space bothers me...

Also I think you should put it in braces? E.g.
```
 > pgbot praise @DWu39 5 for apologizing
> @DWu39 Yes! :grinning: (for apologizing)
```